gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -90,9 +91,10 @@ export const wrapCommand = (command) => {
         this.inspectProposalState(proposal, threshold, owners)
         return result
       }
-
       const result = await this.wrapAction(this.executeProposal)(proposal, proposalContext)
-      this.inspectProposalState(proposal, threshold, owners)
+      if (Object.keys(result).length === 0) {
+        logger.error('Execution failed')
+      }
```

Comment:
We are swallowing the error here: https://github.com/smartcontractkit/chainlink-solana/pull/115/files#diff-e137a7e73b6feadf1e43af5a461789ae132405b97460f422e156a88416ddd871R119
Please re-throw it again after logging